api/python/quilt3/packages.py

```diff
@@ -650,10 +655,6 @@ def _walk_dir_meta(self):
     @classmethod
     @ApiTelemetry("package.load")
     def load(cls, readable_file):
```

Comment:
I wonder if this should just be a private method, with no telemetry? I don't think it's particularly useful for a regular user.

Reply:
It's used as a workaround for performance issues in `quilt3`, so I think it is important to measure.